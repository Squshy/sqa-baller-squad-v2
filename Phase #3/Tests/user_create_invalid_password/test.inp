CREATE
BIGBOIDADDYDUDE
HeDoBeVibingForRealTho
BS
exit