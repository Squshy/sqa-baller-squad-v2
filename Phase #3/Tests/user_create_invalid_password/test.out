=========================================================
                        CREATE
=========================================================

Enter a username: BIGBOIDADDYDUDE

Enter a password: HeDoBeVibingForRealTho

---------------------------------------------------------
        Password must be less than 12 characters.
=========================================================






